tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3825,9 +3826,15 @@ public function theAdministratorDisablesTheFollowingUsersUsingTheProvisioningApi
 	 */
 	public function adminHasDisabledUserUsingTheProvisioningApi(?string $user):void {
 		$user = $this->getActualUsername($user);
-		$this->disableOrEnableUser($this->getAdminUsername(), $user, 'disable');
-		$this->theHTTPStatusCodeShouldBeSuccess();
-		$this->ocsContext->assertOCSResponseIndicatesSuccess();
+		if (OcisHelper::isTestingWithGraphApi()) {
+			$response = $this->graphContext->editUserUsingTheGraphApi($this->getAdminUsername(), $user, null, null, null, null, false);
+		} else {
+			$response = $this->disableOrEnableUser($this->getAdminUsername(), $user, 'disable');
+		}
+		Assert::assertEquals(
+			200,
+			$response->getStatusCode()
+		);
```

Comment:
Please, add some nice error message there
```suggestion
		Assert::assertEquals(
			200,
			$response->getStatusCode(),
			"nice error message with response body"
		);
```